_posts/phuong-my_do/2017-04-28-Business_analyst.md

```diff
@@ -0,0 +1,112 @@
+
+# Who are the players in an IT project and interactions with Brian?
+
+### The Business Stakeholders
+They are the product owner (in our context the principal investigator), and the subject matter experts or super-users (the bioinformaticians and lead scientists).
+
+This is the starting point of all projects: what do the stakeholders want? The higher the hierarchy the higher level the requirements. The stakeholder’s role is to give ideas of what they want but Brian is here to dig into the topic and provide more details before any discussion can happen with the software engineering team. 
+
+### The Project Manager (PM)
+Brian is responsible for ensuring requirements get done. The PM ensures the requirements get developed on a schedule.
+
+### The Software Development Team (architects, bioinformaticians, technical leads, developers)
+
+The Dev team may not need Brian, they could go directly to the stakeholder, collect the requirements and implement the solution. In larger-size projects, Brian’s role is to document the requirements or implementation options, track the impacts, log any decision, any back-and-forth (yes, the stakeholders can change their mind in the light of external circumstances). Usually the Software Developers find their joy in prototyping and coding, not so much in writing formalized documentation of the stakeholders expectations.  
+
+### The Quality Assurance Team 
+The QAs perform Quality Assurance based on requirements written by Brian.
+
+### The users (bioinformaticians, researchers, students, the cancer research community). 
+
+Brian will provide assistance in training the users to their new application. For larger projects a user service team could be dedicated; in that case, Brian would provide Level 2 support to the user service Team for a better understanding of the solution.
+   
+**The key in such IT projects is for Brian not only to document but mostly to understand** what bioinformaticians, researchers and software engineers tell him to accurately document it. 
+It’s about asking the right question. To be able to do so, Brian needs to understand the business goals as much as the technical aspects of an application.
+
+Who has never seen this scenario? User Service Team: “The user cannot use the feature, is it a development issue? >> Developper: "No, it’s a data issue" >> Data Quality: "No, it’s an environment or network issue" >> Dev-Ops: "No, the user did not use it properly”. Brian’s role here is to clarify the issue, record everyone’s understanding and challenge the answers to ensure the right solution is identified.
+
+# Brian’s techniques and main activities
+* **Define the 5 W’s**: Who did What, Where, When, and Why: Probably the basics when Brian starts business analysis. Answering What and Why is sometimes challenging in the cancer genomics research field characterized by exploration (“I don’t know exactly what I want but we should keep moving in this direction”)
```

Comment:
I don't really understand this sentence, also some grammar errors

Answering What and Why is sometimes challenging in the cancer genomics research field characterized by exploration (“I don’t know exactly what I want but we should keep moving in this direction”)

maybe instead:

Answering What and Why can be challenging in cancer genomics research, or any field with significant complexity. Instead we choose a direction and follow it, discovering the answers along the way.